cleanlab_studio/internal/api/api.py

```diff
@@ -18,6 +24,39 @@
 cleanset_base_url = f"{base_url}/cleansets"
 
 
+def telemetry(func: Callable[..., Any]) -> Callable[..., Any]:
+    """
+    Decorator to send stack trace to backend if an exception is raised
+
+    Can only use on functions whose first arg is api_key
+    """
+
+    @functools.wraps(func)
+    def tracked_func(api_key: str, *args: Any, **kwargs: Any) -> Any:
+        print("using telemetry")
+        try:
+            result = func(api_key, *args, **kwargs)
+            return result
+        except Exception as err:
+            with contextlib.suppress(Exception):
+                arg_list = [repr(arg) for arg in args]
+                arg_list.extend(f"{k}={v!r}" for k, v in kwargs.items())
+                arg_str = ", ".join(arg_list)
```

Comment:
consider that this is considerably more invasive than just sending the traceback. would recommend checking w Anish to see if this is what we want to do at the moment